lib/serverless/api-gateway.js

```diff
@@ -158,6 +161,25 @@ function isGatewayV2Event(event) {
   const keys = Object.keys(event).sort().join(',')
   return keys === httpApiV2Keys && event?.version === '2.0'
 }
+const albKeys = [
+  'requestContext',
+  'httpMethod',
+  'path',
+  'queryStringParameters',
+  'headers',
+  'body',
+  'isBase64Encoded',
+  'rawHeaders',
+  'multiValueQueryStringParameters',
+  'pathParameters'
+]
+  .sort()
```

Comment:
I recommend listing them in sorted order to avoid the unnecessary sorting operation cost.

Reply:
I'm dispensing with the sort and string conversion, as we're no longer looking for an exact match of all keys.